posts/2022-10-06-experimenting-with-voting-schemes.adoc

```diff
@@ -0,0 +1,400 @@
+= Introduction to e-voting schemes
+agoetschm
+v1.0, 2022-10-06
+:title: Introduction to e-voting schemes
+:lang: en
+:tags: [voting, crypto, zero-knowledge, en]
+
+Given the availability of technology in our society as well as the need for tools supporting democratic decision making, it is not surprising that e-voting is a growing research field.
+But unlike suggested by their name, e-voting schemes are not only digitalized version of traditional voting systems where paper ballots are counted by hand.
+They go a step further by leveraging cryptographic methods to provide privacy and verifiability guarantees. 
+In particular, trust is shifted from the system to independent verifiers who can check that everything went well.
+
+The aim of this post is to gain an understanding of how this type of systems works with a concrete example. We will go over some background information on voting schemes, give a description of the Helios 2.0 scheme, look into a possible attack and finish with a few pointers about key parts of a simplified implementation.
+
+
+== Types of voting schemes
+In traditional voting systems, each voter puts a piece of paper carrying her vote in a box. Some trusted administrator then opens the box, counts the votes and publishes the result.
+
+The class of schemes we are talking about follow the same pattern, except that instead of casting a piece of paper in a box, the vote is encrypted before being shared on a public bulletin board, i.e. a medium where anyone can read from. 
+This allows observers to check some properties of the ballots without seeing their content since they are encrypted.
+The ballots can then be tallied by the organizer in possession of the necessary decryption material.
+
+.Voting process with some details we will get into later.
```

Comment:
Is this a caption on the diagram that follows?

Suggestion: I would replace the ascii art based diagrams with 'real' drawings. Do you have access to a drawing tool such as Omnigraffle? If not, I think we should still convert the drawings and see who could make the drawings for you (there aren't that many in the article anyway).

Reply:
yes it's the caption

you're right proper diagrams should make it a bit cleaner